main.go

```diff
@@ -117,6 +117,12 @@ func handlerStyle(w http.ResponseWriter, rq *http.Request) {
 	}
 }
 
+func handlerToolbar(w http.ResponseWriter, rq *http.Request) {
+	log.Println(rq.URL)
+	w.Header().Set("Content-Type", "text/js;charset=utf-8")
```

Comment:
It should be `text/javascript`, see [these weird docs](https://developer.mozilla.org/ru/docs/Web/HTTP/Basics_of_HTTP/MIME_types#textjavascript)